docs/framework/configure-apps/file-schema/wcf/issuerchannelbehaviors-element.md

```diff
@@ -2,20 +2,21 @@
 title: "<issuerChannelBehaviors> Element"
 ms.date: "03/30/2017"
 ms.assetid: f7378673-8e9b-45b2-98d1-cf5dccdd8c40
+no-loc: [<system.serviceModel>, <behaviors>, <endpointBehaviors>, <behavior>, <clientCredentials>, <issuedToken>, <issuerChannelBehaviors>, <dataContractSerializer>]
 ---
 
 # \<issuerChannelBehaviors> Element
 
 Contains a collection of Windows Communication Foundation (WCF) client endpoint behaviors (defined in the configuration) to be used when communicating with the specified Service Token Services. The defined behaviors cannot include any [\<clientCredentials>](clientcredentials.md) elements.
 
-[**\<configuration>**](../configuration-element.md)\
-&nbsp;&nbsp;[**\<system.serviceModel>**](system-servicemodel.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;[**\<behaviors>**](behaviors.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<endpointBehaviors>**](endpointbehaviors.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<behavior>**](behavior-of-endpointbehaviors.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<clientCredentials>**](clientcredentials.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<issuedToken>**](issuedtoken.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**\<issuerChannelBehaviors>**  
+[\<configuration>](../configuration-element.md)\
```

Comment:
Although I don't have any strong feeling about this change, it is true that the majority of articles have boldfaced the configuration elements in this section.

Reply:
I think I can do an easy search and replace on existing articles. I was just feeling that the bold was a bit too strong in this section.